packages/wallet/test/integration/cip30mapping.test.ts

```diff
@@ -106,6 +106,64 @@ describe('cip30', () => {
   });
 
   // almost as good as running in browser
+
+  describe('confirmation callbacks', () => {
+    let api: WalletApi;
+    beforeAll(() => {
+      let onMessageHandler: any;
+      const chromeRuntime = (global as any).chrome.runtime;
+      chromeRuntime.onMessage.addListener.mockImplementation((handler: any) => (onMessageHandler = handler));
+      chromeRuntime.sendMessage.mockImplementation((_: any, message: any, callback: any) =>
+        onMessageHandler(message, null, callback)
+      );
+      cip30.initialize(
+        wallet,
+        jest
```

Comment:
The tests would be clearer and less brittle if you set it to a variable, e.g.
```ts
describe('confirmation callbacks', () => {
  let confirmationCallback: jest.Mock;
  ...
```
Then you can set `mockResolvedValue` within the tests instead of beforeAll